### ISSUE TITLE
The logging setup by utils.display only ever uses error/info log levels.

### ISSUE THREAD

**Issue by alikins** [ansible/ansible#25761]
##### SUMMARY

Any log records sent to the logger created in display only use error or info.
'warning' and 'debug' could be used for finer grained logs, especially if combined
with additional log handlers.

Additionally, the ansible specific verbosity levels could be mapped to custom log levels
('v', 'vvv', etc). 

The additional verbosity levels ansible/display already use could be passed into display.display and/or its logger to allow finer grained log configuration.

##### ISSUE TYPE

 - Bug Report
 
##### COMPONENT NAME

lib/ansible/utils/display.py

##### ANSIBLE VERSION

```
ansible 2.4.0 (devel 44687bb917) last updated 2017/06/15 13:12:13 (GMT -400)
  config file = None
  configured module search path = [u'/home/adrian/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/adrian/src/ansible/lib/ansible
  executable location = /home/adrian/src/ansible/bin/ansible
  python version = 2.7.13 (default, May 10 2017, 20:04:28) [GCC 6.3.1 20161221 (Red Hat 6.3.1-1)]

```

##### STEPS TO REPRODUCE

```yaml

```

##### ACTUAL RESULTS

```

```

**Comment by ssbarnea** [comment]
I think that Ansible should be gradually refactored to use the standard Python logging method instead of the `Display()` facility as that one provides everything needed for extending logging. Callback plugins should work more like Logging handlers.

Current logging design (implementation) is crippled quite well, harcoded logging levels, impossible to define different logging levels for each destination and so on. My impression that that the Display() class is the root of all evil on Ansible logging issues... just one example: it does format the message before knowing where the message will be sent.... is like HTML before CSS was introduced ;)

**Comment by bcoca** [comment]
closed via #56311
